sdk/src/azure/core/az_json_builder.c

```diff
@@ -9,12 +9,66 @@
 
 #include <azure/core/_az_cfg.h>
 
-static AZ_NODISCARD az_span _get_remaining_span(az_json_builder* json_builder)
+AZ_NODISCARD az_result az_json_builder_init_chunked(
+    az_json_builder* json_builder,
+    az_span first_destination_buffer,
+    az_span_allocator_fn allocator_callback,
+    az_allocator_context* context,
```

Comment:
void* user_context - this parameter name should match the field name in the allocator context struct